ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -296,6 +296,7 @@ sortFieldExpression
 
 fieldExpression
     : qualifiedName
+    | highlightFunction
```

Comment:
PPL fields command only support qualifiedName, which is different with SQL SELECT Actually, I did not see strong reason to extend it to support highlight function in there. 
```
fieldsCommand
    : FIELDS (PLUS | MINUS)? fieldList
    ;
```

What is user query with `fields - highlight(title)`?

Reply:
`fields - highlight(title)` results in a `SemanticCheckException`

Reply:
1. if we change the fields grammer, which is a breaking chagne.
2. if we add highlight, how would we expected user use it? `source=beer | where multi_match([Title, Body, Tags], 'taste') |  highlight('Body'), highlight('T*')` The response is HTML. which means this feature is coupled with frontend, right?

Reply:
PPL will be moved to separate PR.